values/src/test/java/org/spine3/time/LocalDatesShould.java

```diff
@@ -39,20 +39,160 @@ public void have_private_constructor() {
     @Test
     public void obtain_current_date() {
         final LocalDate today = LocalDates.now();
-
         final Calendar calendar = Calendar.getInstance();
+
         assertEquals(calendar.get(Calendar.YEAR), today.getYear());
         assertEquals(calendar.get(Calendar.MONTH) + 1, today.getMonthValue());
         assertEquals(calendar.get(Calendar.DAY_OF_MONTH), today.getDay());
     }
 
     @Test
+    public void obtain_date_from_year_month_day_values() {
+        final int year = 1976;
+        final int month = 3;
+        final int day = 1;
+        final LocalDate birthday = LocalDates.of(1976, MonthOfYear.APRIL, 1);
+        final Calendar calendar = Calendar.getInstance();
+        calendar.set(year, month, day);
+        assertEquals(calendar.get(Calendar.YEAR), birthday.getYear());
+        assertEquals(calendar.get(Calendar.MONTH) + 1, birthday.getMonthValue());
```

Comment:
Once again, just use `getMonth(calendar)`, which embeds the knowledge of the `+1` difference in its body. DRY.